txtl-liposome_thin-film.md

```diff
@@ -0,0 +1,109 @@
+# TX-TL Liposome Using Thin Film Hydration
+
```

Comment:
(about title) actually the procedure describes the preparation of simple vesicles containing just buffer and possibly a dye. In the case of TX-TL some steps cannot be applied (e.g., heating 60 degrees for 1 h...)

Reply:
@PaolaTorre or @aaronengelhart: Can one of you suggest a change to address @p-stano's comment? I can implement once I know what to change.  

I think the idea was that our v0.1 protocols should be able to express TX-TL?  Can we/should we modify this to allow that?  Or leave out for 0.1 until we get some feedback from testers?

Reply:
@murrayrm I agree the protocol needs to be tested without TX/TL and then with TX/TL.

Reply:
In our experience heating is not necessary with pure popc or popc/2chol. I think we can omit in testing.

Reply:
Leaving unchanged for now.